Chess-On-Chain.md

```diff
@@ -0,0 +1,255 @@
+
+
+
+
+
+
+
+
+
+### Ecosystem Fit
+We aspire to be a parachain with polkadot/kusama
+
+
+
+## Team :EncryptedClan
+### Team members
+Architect: Teke George
+Product Manager: Tari Dekpen
+Runtime Developer: Teke George 
+UI Development : Eric Ebikabowei
+Smart Contract Developer: Gift Moris
+
+
+### Contact
+- **Contact Name:Teke George Seidougha
+- **Contact Email:Georgeteke45@gmail.com
+- **Website:IncryptoEncrypted.org
+
+### Legal Structure
+
+- **Registered Address:** Blockchain Hub okutukutu yenagoa, Bayelsa State , Nigeria
+- **Registered Legal Entity:** EncryptedClan ltd
+
+### Team's experience
+
+We are a team of builders that have been building together for 7 years together , on web2/web3  builds , we're focused on RUST blackchains  and networks like STACKS, SOLANA , NEAR ,SUBSTRATE etc
+we Anchored the first substrate training in nigeria with Octpous network . We are the first and only blockchain hub in bayelsa , we've gotten a grant from STACKS blockchain in the past , and we've anchored several hackathons and bootcamps .
+
+### Team Code Repos
+- https://github.com/InCryptoEncrypted/ChessOnChain
+
+- https://github.com/Teke-pixel
+- https://github.com/taridekpen
+- https://github.com/erricafes
+- https://github.com/zeus-adin
+
+
+## Development Status :open_book:
+ 
+ https://chess.incryptoencrypted.org/
+ 
+ https://github.com/InCryptoEncrypted/ChessOnChain
+
+Development Roadmap 
+Overview
+Full-time equivalent (FTE): Milestone 1 , 2
+Total Costs: 30k $
+
+
+
+
+
+Milestone 1 Example — Functional Pallets 
+Estimated duration: 3 month
+
+FTE: 1,5
+
+Costs: 20,000 USD
+
+exclamation The default deliverables 0a-0d below are mandatory for all milestones, and deliverable 0e at least for the last one.
+
+| Number |Deliverable  | Specification |
+| -------- | -------- | -------- |
+|0a     | License   | Apache 2.0 / GPLv3 / MIT / Unlicense   |
+| 0b   | Documentation | We will provide a repository with  documentation of the pallets and a basic "How TO Guide " that explains how a user can (for example) spin up one of our Substrate nodes and earn tokens from gameplay , Step by Step guide to spawn up wallets , play chess and Earn tokens          |
+| 0c  |Testing and Testing out  |The code will have unit-test coverage  to ensure functionality . In the guide, we will describe how to run these tests   |
+| 0d   | Docker  :      | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone.    |
+|  0e  | Article :  |  We will write an article on each pallet and build in this milestones    |
+|  1   | OFC Pallet:  | implementation of  offchain workers Pallet to get data from Game played on ChessOnchain As JSON files with http requests  Offchain worker pallet  will collect Game data such as moves , winning streaks ,ranking and rating |
```

Comment:
This is the first mention of an offchain worker. Can you provide details about what this pallet does, where the data comes from and where it goes?